backend/src/main/java/com/exe01/backend/dto/request/account/BaseAccountRequest.java

```diff
@@ -0,0 +1,40 @@
+package com.exe01.backend.dto.request.account;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class BaseAccountRequest {
+    @NotBlank(message = "This field must not be blank")
+    @NotNull(message = "This field must not be null")
+    @Size(max = 100, message = "Username must be less than or equal to 100 characters")
+    private String username;
+
+    @NotBlank(message = "This field must not be blank")
+    @NotNull(message = "This field must not be null")
+    @Size(max = 100, message = "Password must be less than or equal to 100 characters")
```

Comment:
– Must be at least 8 characters
– Must include one lowercase character
– Must include one uppercase character
– Can't be too common